### PR TITLE
Update library for latest RN versions

### PR DESCRIPTION
**Pull request by a-morenovalls**

### CODE REVIEW

File: src/OptionButton.js [hunk 1]
```diff
@@ -13,10 +18,10 @@ type Props = {
   style?: any,
   textStyle?: any,
   separatorStyle: any
-}
-type State = void
+};
+type State = void;
```
Comment (alvaromb) [inline src/OptionButton.js]: You can remove this type, and also form the component annotation.

**Review by alvaromb — CHANGES_REQUESTED** [review]

File: src/OptionButton.js (hunk 1 above)
Comment (a-morenovalls) [reply src/OptionButton.js]: Removed!

**Comment by alvaromb** [comment]
LGTM

**Comment by alvaromb** [comment]
@Moreno97 feel free to release.

**Comment by a-morenovalls** [comment]
> @Moreno97 feel free to release.

I can't publish the latest version on NPM, it's owned by you.

**Pull request merged**